spec-main/api-autoupdater-darwin-spec.ts

```diff
@@ -0,0 +1,263 @@
+import { expect } from 'chai'
+import * as cp from 'child_process'
+import * as http from 'http'
+import * as express from 'express'
+import * as fs from 'fs-extra'
+import * as os from 'os'
+import * as path from 'path'
+import { AddressInfo } from 'net';
+
+const features = process.atomBinding('features')
+
+const fixturesPath = path.resolve(__dirname, '../spec/fixtures')
+
+// We can only test the auto updater on darwin non-component builds
+const describeFn = (process.platform === 'darwin' && !process.mas && !features.isComponentBuild() ? describe : describe.skip)
+
+describeFn('autoUpdater behavior', function () {
+  this.timeout(120000)
+
+  let identity = ''
+
+  beforeEach(function () {
+    const result = cp.spawnSync('bash', [path.resolve(__dirname, '../script/codesign/get-trusted-identity.sh')])
```

Comment:
i don't believe you need `bash` here if the .sh is `+x`

Reply:
@nornagon You'd think so, but it worked on CI without `bash` and blew up locally.  This seemed like a reasonable option to ensure it worked everywhere.

Reply:
blew up how? that seems relevant

Reply:
`ENOENT` iirc